config/routes.rb

```diff
@@ -69,5 +69,6 @@
   resources :course_reserves, only: :index, path: "reserves"
 
   get 'article/home' => 'article#new' # alias for article search home page
-  resources :article, only: %i[index new show], defaults: { format: :html }
```

Comment:
@jkeck @drh-stanford I ran into errors when trying to use article_home_path in tests. I noticed our other routes don't have `format:html` specified. Is it ok to remove this?